source/_pages/privacy-policy.blade.php

```diff
@@ -19,24 +19,91 @@
     <section class="ud-blog-details">
       <div class="container">
         <div class="row">
-          <div class="col-lg-8">
+          <div class="col-12">
```

Comment:
I think that would be best create this page as markdown to be possible translate entire text and not separate paragraphs.